Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalIsoTrk_Output_cff.py

```diff
@@ -11,14 +11,17 @@
     outputCommands = cms.untracked.vstring( 
         'keep *_IsoProd_*_*',
         'keep *_TkAlIsoProd_*_*',
+        'keep *_hbhereco_*_*',
```

Comment:
@mmusich I can have a look. Which of [these input datasets](https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/Configuration/PyReleaseValidation/python/relval_steps.py#L190-L246) is best suited for such a test?